quipucords/quipucords/settings.py

```diff
@@ -201,9 +201,18 @@
 LOGGING_HANDLERS = os.getenv('DJANGO_LOG_HANDLERS', 'console').split(',')
 VERBOSE_FORMATTING = '%(levelname)s %(asctime)s %(module)s ' \
     '%(process)d %(thread)d %(message)s'
-DEFAULT_LOG_FILE = os.path.join(BASE_DIR, 'app.log')
+
+LOG_DIRECTORY = os.getenv('LOG_DIRECTORY', BASE_DIR)
+DEFAULT_LOG_FILE = os.path.join(LOG_DIRECTORY, 'app.log')
 LOGGING_FILE = os.getenv('DJANGO_LOG_FILE', DEFAULT_LOG_FILE)
 
+DEFAULT_SCAN_DATA_LOG = os.path.join(LOG_DIRECTORY, 'scan_data_log')
+SCAN_DATA_LOG = os.getenv('SCAN_DATA_LOG', DEFAULT_SCAN_DATA_LOG)
```

Comment:
Again just for consistency maybe better as `SCAN_DATA_LOG_FILE`.

Reply:
Done.